toolchains/docker.gyselalibxx_env/tests_toolchain.cmake

```diff
@@ -1,6 +1,9 @@
 
 include(${CMAKE_CURRENT_LIST_DIR}/../cicd_default_toolchain.cmake)
 
+# CMake options
+set(CMAKE_BUILD_TYPE FakeDebug)
```

Comment:
Why `FakeDebug` instead of real `Debug`?

Reply:
I was not sure we wanted `-g` but I am fine with `Debug`

Reply:
Done in 769769005651c0031ce9aff248b068fedac82ced